gemrb/core/Scriptable/Actor.cpp

```diff
@@ -6233,6 +6233,12 @@ bool Actor::ValidTarget(int ga_flags, const Scriptable *checker) const
 			if (Modified[IE_CHECKFORBERSERK]) return false;
 		}
 	}
+	if (ga_flags & GA_BUMPING) {
+		if (checker->Type != ST_ACTOR) return false;
+		if ((!((Actor*)(checker))->IsPartyMember() || this->GetStat(IE_EA) >= EA_EVILCUTOFF) && ((Actor*)(this))->GetStat(IE_NPCBUMP) < 1) {
```

Comment:
Please split this into two if checks. And 
```suggestion
		if ((!((Actor*)(checker))->IsPartyMember() || this->GetStat(IE_EA) >= EA_EVILCUTOFF) && GetStat(IE_NPCBUMP) < 1) {
```